src/app/layout/toolbar/toolbar-user/toolbar-user.component.scss

```diff
@@ -0,0 +1,72 @@
+.toolbar-user-container {
```

Comment:
class name should match component name. BTW you can call it smth like LoginStatus or UserButton